accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,502 @@
+    public SkiaSharpImage(Android.Media.Image nativeImage) { /* ... */  }
+#endif
+
+#if IOS
+    public SkiaSharpImage(NSImage nativeImage) { /* ... */ }
+#endif
+
+#if WINDOWS
+    public SkiaSharpImage(Windows.Media.BitmapImage nativeImage) { /* ... */ }
+#endif
+}
+```
+
+## Requirements
+
+### Goals
+
+* Use naming that aligns with product strategy
+* Merge .NET Core and .NET Standard into a single concept
+* Developers should be able to understand compatibility relationships without
+  having to consult a mapping table.
+* Provide compatibility with existing concepts and NuGet packages
+* It would be great to get this into .NET 5 Preview 1
+
+### Non-Goals
+
+* Replace TFMs or expand runtime identifiers (RIDs)
+* Support multi-targeting between OS versions
+
+## Design
+
+| TFM             | Compatible With                                            | Comments                          |
+|-----------------|------------------------------------------------------------|-----------------------------------|
+| net5.0          | net1..4 (with NU1701 warning)                              | No WinForms or WPF                |
+|                 | netcoreapp1..3.1 (warning when WinForms/WPF is referenced) |                                   |
+|                 | netstandard1..2.1                                          |                                   |
+| net5.0-android  | xamarin.android                                            |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-ios      | xamarin.ios                                                |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-macos    | xamarin.macos                                              |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-tvos     | xamarin.tvos                                               |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-watchos  | xamarin.watchos                                            |                                   |
+|                 | (+everything else inherited from net5)                     |                                   |
+| net5.0-win      | netcoreapp1..3.1                                           | WinForms + WPF and probably WinUI |
+|                 | (+everything else inherited from net5)                     |                                   |
+| Tizen, Unity... | Will follow the Xamarin model                              |                                   |
+
+***Open Issue**. In case of Xamarin, there is a discussion of whether the
```

Comment:
```suggestion
**Open Issue**. In case of Xamarin, there is a discussion of whether the
```

Reply:
Not a bug. That's for italics. The closing star is at the end of the paragraph 😄

Reply:
Maybe try with _? GitHub is confused with this formatting and can't render the rest of the text right in preview.

Reply:
Ah, regex syntax coloring FTW! I'll try.